src/Symfony/Component/HttpFoundation/HeaderBag.php

```diff
@@ -124,7 +124,7 @@ public function get(string $key, string $default = null)
      * @param string|string[] $values  The value or an array of values
      * @param bool            $replace Whether to replace the actual value or not (true by default)
      */
-    public function set(string $key, $values, bool $replace = true)
+    public function set(string $key, string|array|null $values, bool $replace = true)
```

Comment:
Are we sure that `null` is acceptable here? If yes, the `@param` annotation is incomplete.

Reply:
I confirm, at least there are tests covering that